plugins/woocommerce/src/StoreApi/Schemas/V1/CheckoutSchema.php

```diff
@@ -373,7 +373,7 @@ public function validate_additional_fields( $fields, $request ) {
 
 			// Only allow custom validation on fields that pass the schema validation.
 			if ( true === $result ) {
-				$result = $this->additional_fields_controller->validate_field( $key, $field_value, $properties[ $key ], $request );
+				$result = $this->additional_fields_controller->validate_field( $key, $field_value, $request, null );
```

Comment:
Could make $address_type optional instead of requiring `null`. Set it to `$address_type = null` in the method itself.

Reply:
ok nice idea will do!

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/44012/commits/75b0950819878c647c6283ea94534075a0372e38